src/api/EscargotPublic.cpp

```diff
@@ -2048,6 +2068,15 @@ bool ContextRef::initDebugger(const char* options)
 #endif /* ESCARGOT_DEBUGGER */
 }
 
+bool ContextRef::isDebuggerRunning()
+{
+#ifdef ESCARGOT_DEBUGGER
+    return !!toImpl(this)->debugger();
```

Comment:
It seems that we should check if `Debugger::m_enabled` value is true or not too.
Debugger might be created but disabled during execution.

Reply:
you are right!